llmfoundry/models/layers/attention.py

```diff
@@ -640,6 +675,9 @@ def forward(
             dropout_p=self.attn_dropout_p,
             training=self.training,
             needs_weights=needs_weights,
+            query_attention_mask_in_length=query_attention_mask_in_length,
+            key_attention_mask_in_length=key_attention_mask_in_length,
+            should_repeat_kv_for_gqa= not is_flash_v2_installed(),
```

Comment:
```suggestion
            should_repeat_kv_for_gqa=not is_flash_v2_installed(),
```

did lint not complain about this?
looks weird to me, but yeah do what the linter says.

Reply:
I manually turned off lint while committing. Sorry about that :D . Will run it